server/datastore/mysql/migrations/tables/20241209164540_AddSecretVariablesTable.go

```diff
@@ -19,8 +19,9 @@ func Up_20241209164540(tx *sql.Tx) error {
 		id INT UNSIGNED NOT NULL AUTO_INCREMENT,
 		name VARCHAR(255) NOT NULL,
 		value BLOB NOT NULL, -- 64KB max value size
-		created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
-		updated_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+		-- Using DATETIME instead of TIMESTAMP to prevent future Y2K38 issues, since updated_at is used as a trigger to resend profiles
```

Comment:
Should this be a new pattern? Or is this just for this one case?

Reply:
DATETIME uses more storage than TIMESTAMP. We can discuss in backend sync.

Reply:
OK, is this change for the next PR?

Reply:
Yes, I just wanted to get it in since I wasn't sure if the next PR would make the cut/release.